heron/api/src/java/org/apache/heron/api/Config.java

```diff
@@ -480,11 +480,19 @@ private static List getRegisteredKryoSerializations(Map<String, Object> conf) {
   }
 
   public static void registerKryoSerialization(Map<String, Object> conf, Class klass) {
+    // Set to Kryo Seralizer when this function is caled. In Storm API, Kryo seralizer is
+    // the default but it is not in Heron. This implicit set could make migration easier
+    // since it is more consistent.
+    setSerializationClassName("org.apache.heron.api.serializer.KryoSerializer");
```

Comment:
create a final field for this `"org.apache.heron.api.serializer.KryoSerializer"` string and reuse it everywhere

Reply:
kk. done.